docs/src/main/sphinx/udf/introduction.md

```diff
@@ -14,8 +15,9 @@ plugin. Details are available in the [developer guide](/develop/functions).
 (udf-declaration)=
 ## UDF declaration
 
-Declare the UDF with a [](/udf/function) keyword using the supported statements
-for [](/udf/sql).
+Declare the UDF with the SQL [](/udf/function) keyword, additional parameters
+such as `RETURNS` or `LANGUAGE` and use the supported statements for [](/udf/sql) or
```

Comment:
This sentence is a bit hard to read. What if we simplify and combine with below

```markdown
Declare the UDF using the [](/udf/function) declaration. [...]

Reply:
Done